plugins/tile/tile-plugin.cpp

```diff
@@ -271,6 +272,48 @@ class tile_output_plugin_t : public wf::pointer_interaction_t, public wf::custom
         });
     };
 
+    wf::key_callback on_toggle_tile_current_ws = [=] (auto)
+    {
+        auto last_focused_view = wf::get_core().seat->get_active_view();
```

Comment:
We need to check that we can use the simple-tile plugin (to avoid for example triggering this while another plugin  is active, like expo or move-drag, changing the window during those operations could result in invalid compositor state. See how this is done in `conditioned_view_execute`, `especially the can_activate_plugin` check.